lib/Drupal/drupal_helpers/Rules.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\drupal_helpers;
+
+use Exception;
+
+/**
+ * Class Rules.
+ *
+ * @package Drupal\drupal_helpers
+ */
+class Rules {
+
+  /**
+   * Set Active.
+   *
+   * Set the active property for a rules configuration.
+   *
+   * @param string $rule_name
+   *   Machine name of the Rule.
+   * @param bool $value
+   *   Set rule active property to value.
+   *
+   * @throws \Exception
+   */
+  public static function setActive($rule_name, $value) {
```

Comment:
the name is misleading. it looks like this should be called `changeState` as it can set a rule to be active or inactive.

I suggest renaming this function and making it protected and exposing 2 public functions `activate` and `deactivate` OR `setActive` and `setInactive`.